ports/rp2/mpnetworkport.c

```diff
@@ -64,7 +64,7 @@ static void gpio_irq_handler(void) {
 }
 
 void cyw43_irq_init(void) {
-    gpio_add_raw_irq_handler_with_order_priority(IO_IRQ_BANK0, gpio_irq_handler, CYW43_SHARED_IRQ_HANDLER_PRIORITY);
+    gpio_add_raw_irq_handler_with_order_priority(CYW43_PIN_WL_HOST_WAKE, gpio_irq_handler, CYW43_SHARED_IRQ_HANDLER_PRIORITY);
```

Comment:
I think this is a bug fix from the sdk.

Reply:
OK.  I wonder how this worked prior to this fix...

Reply:
pass, but we had the same bug in the sdk for awhile https://github.com/raspberrypi/pico-sdk/pull/1069